.github/workflows/update-performance-dispatch.yml

```diff
@@ -0,0 +1,23 @@
+name: Update performance dispatch
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'polaris-react/**'
+
+jobs:
+  update-performance-dispatch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v6
+        with:
+          github-token: ${{ secrets.SHOPIFY_GH_ACCESS_TOKEN }}
```

Comment:
There is also `secrets.GITHUB_TOKEN`, not sure which one or if it matters

Reply:
I think this is correct https://github.com/Shopify/polaris/pull/7331/files

Reply:
GITHUB_TOKEN is the token provided by github for doing things. I believe it has as access to stuff within this repo but not outside of it. So you do want SHOPIFY_GH_ACCESS_TOKEN - which is provided by Shopify's token rotator thingy. You might need to add polaris-coverage-shopifycloud-com to the list of repos that token has access to at [here](https://github.com/Shopify/github-actions-access-provider/blob/main/.github/workflows/polaris-team-polaris.yml#L14)